packages/lesswrong/components/localGroups/LocalGroupsList.tsx

```diff
@@ -3,15 +3,19 @@ import { registerComponent, Components } from '../../lib/vulcan-lib';
 import { useMulti } from '../../lib/crud/withMulti';
 import { createStyles } from '@material-ui/core/styles'
 
+export const loadMoreSectionFooterStyles = theme => ({
```

Comment:
I think we should try to avoid sharing css this way. I don't like the way that often in codebases, you want to change some CSS, and then you need to check a bunch of places to see that you haven't broken anything. The way that the react ecosystem solves that is by tying css to components.

Obviously some things like flexGrow shouldn't be tied to the component. But I would just copy and paste those.

I'm not sure about the purpose of the pseudo-element styles, but I wonder if they can be put on the component?

If you really want to share this CSS, but not put it on the component, please move this declaration to the LoadMore component.

Reply:
Hmm. Currently our paradigm seemed to be "if you're going to copypaste some CSS  from one place to another, better to export it this way so things don't get out of sync."

I do think that doing this is usually a warning flag that your component isn't architected right (and I agree it is in this case). I went and changed it. I'm not sure whether I endorse making this sort of incremental improvement is worth the slowdown of random PRs that were mostly not about this.

(The status quo is that there were several different components that had a similar style, and it seemed to me that getting them synced up was a marginal improvement. I went and consolidated all of them into the LoadMore component, which wasn't really the point of this PR, which wasn't a very important PR to begin with, but seems like this part of the codebase was due for a cleanup ayway)